core/ui/src/main/kotlin/de/tum/informatics/www1/artemis/native_app/core/ui/markdown/MarkdownText.kt

```diff
@@ -224,11 +227,16 @@ fun createMarkdownRender(context: Context, imageLoader: ImageLoader?, imageWith:
             CoilImagesPlugin.create(
                 object : CoilImagesPlugin.CoilStore {
                     override fun load(drawable: AsyncDrawable): ImageRequest {
+                        var height = 800
+                        if (drawable.destination.contains("android.resource://de.tum.cit.aet.artemis/")) {
+                            height = 52
```

Comment:
Magic numbers: please create constants for `DEFAULT_IMAGE_HEIGHT` and `LINK_TYPE_HINT_ICON_HEIGHT` (or what ever name fits better)